src/pages/LandingPage.vue

```diff
@@ -57,12 +59,11 @@ export default {
     }
   },
   computed: {
-    carouselItems() {
-      return this.shopItems.filter(si => si.favorite === true)
-    },
+    ...mapState({shopItemsStore: 'shopItems'}),
+    ...mapGetters({carouselItems: 'carouselPhotos'}),
   },
   async mounted (){
-    this.getUpdatedCarouselItems()
+    this.$store.dispatch('getCarouselPhotos', {path: '/api/shop'})
```

Comment:
Consider using `mapActions` to wire in the actual function of the store instead of `$store.dispatch`. 😉  
Also, do you really need explicit `api/shop` parameter? Shouldn't store handle it inside? 🤔